misc/discoverynode/README.md

```diff
@@ -8,5 +8,13 @@
 `~/venv/bin/python3 -m pytest tests/`
 
 ## Integration test (no MQTT)
-`sudo testing/integration/integration_test.sh`
+
+```
+sudo testing/integration/integration_test.sh
+
+sudo docker ps -a | grep "discoverynode-test-" | awk '{print $1}' | sudo xargs docker stop
+
+docker run --network=test-network -v  $PWD/dump:/data --rm -ti travelping/pcap
```

Comment:
Maybe add a specific container name?

You don't need "-ti" if there's no terminal attached.